### PR TITLE
CORE-9732 - remove-contextLogger

### PR DESCRIPTION
**Pull request by driessamyn**
Removed use of `contextLogger` and `loggerFor` as they should not be in the API lib.

API change: https://github.com/corda/corda-api/pull/813

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -40,7 +40,7 @@ bouncycastleVersion=1.72
 # Corda API libs revision (change in 4th digit indicates a breaking change)
 # Change to 5.0.0.xx-SNAPSHOT to pick up maven local published copy
 #cordaApiVersion=5.0.0.xxx-SNAPSHOT
-cordaApiVersion=5.0.0.626-beta+
+cordaApiVersion=5.0.0.627-alpha-1674749342997
```
Comment (driessamyn) [inline gradle.properties]: Will be replaced when API PR approved and built.

**Review by lankydan — DISMISSED** [review]
Browsed through some and changes look fine.

General question moving forward, do we want to stick to

```
LoggerFactory.getLogger(this::class.java.enclosingClass)
```

Or do something like

```
LoggerFactory.getLogger(MyClass::class.java)
```

I appreciate we didn't do this here because it would be an impossible find and replace to execute.

**Comment by driessamyn** [comment]
> Browsed through some and changes look fine.
> 
> General question moving forward, do we want to stick to
> 
> ```
> LoggerFactory.getLogger(this::class.java.enclosingClass)
> ```
> 
> Or do something like
> 
> ```
> LoggerFactory.getLogger(MyClass::class.java)
> ```
> 
> I appreciate we didn't do this here because it would be an impossible find and replace to execute.

I don't really have a strong opinion either way, but indeed, choose the option that was possible for me to do a global find/replace.

**Review by vkolomeyko — DISMISSED** [review]
LGTM

**Review by dimosr — DISMISSED** [review]

**Comment by driessamyn** [comment]
Green build with Alpha version: https://ci02.dev.r3.com/blue/organizations/jenkins/Corda5%2Fcorda-runtime-os/detail/PR-2996/12/pipeline

**Review by vkolomeyko — APPROVED** [review]
LGTM

**Pull request merged**